### PR TITLE
fix(utils): fix incorrect example in docs & add test code

### PR DESCRIPTION
**Pull request by ChanhyukPark-Tech**
## Overview

- Docs example for decommaizeNumber was given as an example for commaizeNumber. 
  - [fix: @toss/utils decommaizeNumber docs](https://github.com/toss/slash/commit/1e5d1f716d5e1943dcf6f900bc80a2d9775cc1bc)
- There are no test codes created for decommaizeNumber, so I wrote some.
  -   [feat: @toss/utils write decommaizeNumber test code](https://github.com/toss/slash/commit/e7c04c1abb1b8599d96dd99349e123112394fb12)

## PR Checklist

- [x] I read and included theses actions below

1. I have read the [Contributing Guide](https://github.com/toss/slash/blob/main/.github/CONTRIBUTING.md)
2. I have written documents and tests, if needed.

### CODE REVIEW

File: packages/common/utils/src/Numbers.spec.ts [hunk 1]
```diff
@@ -37,6 +38,11 @@ describe('Numbers', () => {
     expect(commaizeNumber('100000000')).toEqual('100,000,000');
   });
 
+  test('decommaizeNumber', () => {
+    expect(decommaizeNumber('13,209,802')).toEqual(13209802);
+    expect(decommaizeNumber('1234,1,,234')).toEqual(12341234);
```
Comment (HoseungJang) [inline packages/common/utils/src/Numbers.spec.ts:43]: Fix this please.

```typescript
expect(decommaizeNumber('12,341,234')).toEqual(12341234);
```

**Review by HoseungJang — CHANGES_REQUESTED** [review]

File: packages/common/utils/src/Numbers.spec.ts (hunk 1 above)
Comment (ChanhyukPark-Tech) [reply packages/common/utils/src/Numbers.spec.ts:43]: I wanted to show that it works well even if there are multiple comma in a string, so is there no need to verify it?🙋🏼‍♂️

Comment (raon0211) [reply packages/common/utils/src/Numbers.spec.ts:43]: I think this is not our intended behavior (we expect only string literals which are correctly commaized), so I think excluding this from our test case is better. (Test cases should only include what we intend the function to do.)

Comment (ChanhyukPark-Tech) [reply packages/common/utils/src/Numbers.spec.ts]: Thank you for your answer!
Only the test code was wrote to verify the intention of the function author.
It was an opportunity to think about it again Thank you so much!

**Review by HoseungJang — APPROVED** [review]
Thanks @ChanhyukPark-Tech  @raon0211 . Next time I will comment with more details.

**Pull request merged**